graphql-introspection-query/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "graphql-introspection-query"
+version = "0.1.0"
```

Comment:
Hmmm it should probably be independent from the rest, since it's its own thing (same argument as for putting it into a different repo). What do you think?

Reply:
I think this is an administrative issue and not a big issue, so you should be easier to manage. :)